InAppSettingsKitFramework/Info.plist

```diff
@@ -15,7 +15,7 @@
 	<key>CFBundlePackageType</key>
 	<string>FMWK</string>
 	<key>CFBundleShortVersionString</key>
-	<string>1.0</string>
+	<string>2.7.2</string>
 	<key>CFBundleSignature</key>
```

Comment:
You're right. I've undone these changes. Probably it's better just to set the Podspec version and a git tag.